closed/OpenJ9.gmk

```diff
@@ -453,15 +459,34 @@ ifneq (,$(OPENJ9_DEVELOPER_DIR))
   CUSTOM_COMPILER_ENV_VARS += DEVELOPER_DIR="$(OPENJ9_DEVELOPER_DIR)"
 endif
 
+
+ifeq (true,$(OPENJ9_ENABLE_CMAKE))
+$(OUTPUTDIR)/vm/cmake.stamp:
+	cd $(OUTPUTDIR)/vm && \
+	$(CMAKE) \
+	  -C $(OPENJ9_TOPDIR)/runtime/cmake/caches/$(OPENJ9_BUILDSPEC).cmake \
+		-DJ9VM_OMR_DIR=$(OPENJ9OMR_TOPDIR) \
+		-DBUILD_ID=$(BUILD_ID) \
+		-DJAVA_SPEC_VERSION=$(VERSION_MAJOR) \
+		-DOPENJ9_BUILD=true \
+		$(OPENJ9_TOPDIR)
+	touch $(OUTPUTDIR)/vm/cmake.stamp
+
+run-preprocessors-j9: \
+		$(OPENJ9_VM_BUILD_DIR)/omr/OMR_VERSION_STRING \
+		$(OPENJ9_VM_BUILD_DIR)/compiler/jit.version \
+		$(OPENJ9_VM_BUILD_DIR)/include/openj9_version_info.h \
+		$(OUTPUTDIR)/vm/cmake.stamp
+
+else
 run-preprocessors-j9 : stage-j9 \
-		$(OUTPUTDIR)/vm/omr/OMR_VERSION_STRING \
-		$(OUTPUTDIR)/vm/compiler/jit.version \
-		$(OUTPUTDIR)/vm/include/openj9_version_info.h
+		$(OPENJ9_VM_BUILD_DIR)/omr/OMR_VERSION_STRING \
+		$(OPENJ9_VM_BUILD_DIR)/compiler/jit.version \
+		$(OPENJ9_VM_BUILD_DIR)/include/openj9_version_info.h \
```

Comment:
The `\` added here broke it I think

Reply:
Yes, seems to have done it. Running a build to confirm there are no other issues

Reply:
That trailing backslash came back, but it can't stay.

Reply:
Oops, appears I rebased an old branch. Should be resolved